docs/source/how_to_add_a_component.rst

```diff
@@ -0,0 +1,112 @@
+How to Add a Component
+======================
+
+Intro
+-----
+
+To integrate a new component into the dashboard you need to add a folder (e.g.
+`maps`) in `utilities/dashboard/components`. The code that generates your component
+should be divided into three modules:
+
+- plotting module (e.g. `mapplot.py`), containing functions that process
+  a relatively "raw" dataset and returns the statistics to be displayed in the
+  plot, and functions that generate the component's plot.
+
+- `create_data.py`, containing functions that process the data (which may include
+  metadata, such as descriptive text) needed to create your component and returns
+  a dictionary of aggregate data.
```

Comment:
I think it could be confusing to describe the plotting module first. I would start chronologically, i.e. with `create_data.py`

Reply:
I made clearer why I describe the plotting module first (it contains core functions, which are called by the other two modules, and it has to be developed first)